lib/utils.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
```

Comment:
Thanks @SimonLab. yes, I agree we need to _understand_ the _benefits_ of the `'use strict';` directive.